### DISCUSSION TITLE
Sort formatted values in Wave table

### DISCUSSION THREAD

**Discussion by shamilh2o** [h2oai/wave#2218] [Q&A]
Hey team,
I am looking to sort values displayed in a table column. 
I am passing the values as integers and the type of column is defined "number".

> data_type="number"

But if I want to display values in comma separated format as below I have to pass the value as a string. Then the sorting doesn't perform as expected.
![image](https://github.com/h2oai/wave/assets/101506482/bd7c43de-80c2-42ad-9a6b-41e1b1f00b56)

wave version 0.26.2

I am looking to give better feeling to the user with comma separated format for lengthy values same time allowing them to sort it. 
Any existing solution which I missed or suggested workaround for this?

Thanks!

cc: @mturoci

**Comment by mturoci** [comment]
For custom sort (and other custom table actions), use [server side table](https://wave.h2o.ai/docs/examples/table-pagination-sort/) which allows custom sorting, filtering, searching etc.
